### ISSUE TITLE
Feature request: ability to export result

### ISSUE THREAD

**Issue by johannspies** [sasha-alias/sqltabs_tracker#4]
The ability to export results to csv-files (data) or images (graphs) or text files (db-info) would enhance the value of this utility.

**Comment by sasha-alias** [comment]
Agree, getting an output out of the tool is certainly in the plans.
Though it can take some time, so not sure when exactly it will be ready.

**Comment by johannspies** [comment]
On Thu, Jul 02, 2015 at 03:32:49AM -0700, Sasha Aliashkevich wrote:
> Agree, getting an output out of the tool is certainly in the plans.
> Though it can take some time, so not sure when exactly it will be ready.

Thanks for your efforts and reply.

Johann

-- 
Johann Spies                            Telefoon: 021-808 4699
Databestuurder /  Data manager		Faks: 021-883 3691

Sentrum vir Navorsing oor Evaluasie, Wetenskap en Tegnologie
Centre for Research on Evaluation, Science and Technology 
Universiteit Stellenbosch.

The integrity and confidentiality of this email is governed by these terms / Hierdie terme bepaal die integriteit en vertroulikheid van hierdie epos. http://www.sun.ac.za/emaildisclaimer
